README.md

```diff
@@ -157,11 +157,15 @@ INFO Totally checked 20 files, valid: 10, invalid: 10, ignored: 0, fixed: 10
 
 This command serves as assistance for human beings to audit the dependencies license, it's exit code is always 0.
 
-You can also use the `--output` or `-o` to save the dependencies' `LICENSE` files to a specified directory so that
-you can put them in distribution package if needed.
+We also support two flags:
+
+|Flag name|Short name|Description|
+|---------|----------|-----------|
+|`--output`|`-o`|Save the dependencies' `LICENSE` files to a specified directory so that you can put them in distribution package if needed.|
+|`--summary`|`-s`|Based on the template, aggregate all dependency information and generate a `LICENSE` file.|
 
 ```bash
-license-eye -c test/testdata/.licenserc_for_test_check.yaml dep resolve -o ./dependencies/licenses
+license-eye -c test/testdata/.licenserc_for_test_check.yaml dep resolve -o ./dependencies/licenses -s LICENSE.tpl
```

Comment:
Can you also add an section to demonstrate how the `LICENSE.tpl` looks like and what variables it can use?

Reply:
Updated.